style.css

```diff
@@ -0,0 +1,109 @@
+@import url('https://fonts.googleapis.com/css2?family=Rubik:wght@400;500;600;700;800;900&display=swap');
+@import url(./docs-css/typography.css);
+
+
+:root {
+  --primary: #071925;
+  --secondary: #f2f2f2;
+  --accent: #805de2;
+  --font-primary: #afafaf;
+  --font-secondary: #f5f5f5;
+}
+body {
+  margin: 0;
+  box-sizing: border-box;
+  background-color: var(--secondary);
+  font-size: 10px;
+  
+}
+ 
+/* ................||nav bar.................... */
+
+.nav {
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  background-color: var(--primary);
+  color: var(--font-secondary);
+  padding: 0.5rem;
+  z-index: 1;
+  
+  
+}
+.heading {
+  font-size: 24px;
+  line-height: 32px;
```

Comment:
Hey,
You can consider using rems instead of px.
Rest is all good!

Reply:
will do that, thanks for the review.